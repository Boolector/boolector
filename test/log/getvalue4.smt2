(set-option :produce-models true)
(set-logic QF_AUFBV)
(declare-const bv (_ BitVec 32))
(define-fun A () (Array (_ BitVec 4) (_ BitVec 8)) ((as const (Array (_ BitVec 4) (_ BitVec 8))) (_ bv0 8)))
(define-fun f ((x (_ BitVec 8))) (_ BitVec 8) (bvadd x (_ bv1 8)))
(check-sat)
(get-value (A))
(get-value ((store A (_ bv4 4) (_ bv1 8))))
(get-value ((_ bv10 8)))
(get-value (bv (bvnot bv)))
(get-value (f))
(exit)
