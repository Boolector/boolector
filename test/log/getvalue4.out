sat
((A ((as const (Array (_ BitVec 4) (_ BitVec 8))) #b00000000)))
(((store A (_ bv4 4) (_ bv1 8)) (store ((as const (Array (_ BitVec 4) (_ BitVec 8))) #b00000000) #b0100 #b00000000)))
(((_ bv10 8) #b00001010))
(
 (bv #b00000000000000000000000000000000)
 ((bvnot bv) #b11111111111111111111111111111111)
)
((f (lambda  ((_f_x (_ BitVec 8))) (_ BitVec 8)
(bvadd _f_x #b00000001))
))
